var/spack/repos/builtin/packages/mapl/package.py

```diff
@@ -238,7 +238,12 @@ class Mapl(CMakePackage):
     variant("extdata2g", default=True, description="Use ExtData2G")
     variant("pfunit", default=False, description="Build with pFUnit support")
     variant("f2py", default=False, description="Build with f2py support")
-
+    variant(
+        "skip_tests",
+        default="none",
+        multi=True,
+        description="List of unit test case numbers to skip",
```

Comment:
Variants enter the hash, so not the proper place to skip unit tests...

Reply:
Can you propose an alternative? I didn't hear back anything in the Spack Slack about how to achieve this without impacting the hash. Does/can the `check()` function use the Spack build environment in such a way we could do it via environment variable (MAPL_SKIP_TESTS=...)? It might be a bit ugly since I think it would have to go under the compiler config, but it's the only thing I can think of at the moment. :thinking: